cloudbuild-e2e-gce.yaml

```diff
@@ -33,5 +33,5 @@ steps:
 
 logsBucket: gs://opentelemetry-ops-e2e-cloud-build-logs
 substitutions:
-  _TEST_RUNNER_IMAGE: gcr.io/${PROJECT_ID}/opentelemetry-operations-e2e-testing:0.10.0
+  _TEST_RUNNER_IMAGE: gcr.io/${PROJECT_ID}/opentelemetry-operations-e2e-testing:latest
```

Comment:
I think we should pin the actual tag for latest version so it doesn't change under us

Reply:
Oh, was the exact version number intentional ? These images are not released very often.